src/transformers/file_utils.py

```diff
@@ -548,3 +548,18 @@ def wrapper(*args, **kwargs):
             raise ImportError(f"Method `{func.__name__}` requires TF.")
 
     return wrapper
+
+
+def unzip(zip_path: str, dest_dir: str) -> None:
+    with ZipFile(zip_path, "r") as zipObj:
+        zipObj.extractall(dest_dir)
+
+
+def save_json(content: Union[Dict, List], path: str) -> None:
+    with open(path, "w") as f:
+        json.dump(content, f)
+
+
+def load_json(path: str) -> Union[Dict, List]:
+    with open(path, "r") as f:
+        return json.load(f)
```

Comment:
Let's keep these in the conversion script?
I don't think we need to abstract 2 liners you only use in your conversion scripts.